ecosystem/sep-0040.md

```diff
@@ -0,0 +1,198 @@
+  4. Any account holding that asset lays claim to the NFT.
+
+The NFT model involves two levels of indirection in order to minimize on-chain storage:
+
+    +---------------+      +----------+      +---------+
+    |  Stellar NFT  | ---> | metadata | ---> |   NFT   |
+    |  account data |      |   JSON   |      | content |
+    +---------------+      +----------+      +---------+
+
+
+## Specification
+This SEP defines an [issuance model](#issuance-model) that defines how NFT assets are issued on Stellar and a [metadata specification](#nft-metadata-specification) that defines how data about the NFT is captured. The metadata is stored in a way such that the Stellar network guarantee its authenticity.
+
+### Issuance Model
+Standard issuance of NFTs must take the following steps:
+
+  1. Create an account to represent the NFT: `CreateAccountOp(A, <reserve>)`
+  2. Configure the NFT's metadata pointer, as described [below](#nft-account-description): `ManageDataOp("type", "...")`
+  3. Form an asset describing your NFT: `Code:Issuer` (where issuer is `A`)
+  4. Create a distribution account: `B = CreateAccountOp()` and `ChangeTrust("Code:Issuer", <limit>)`
+  5. Issue the tokens: `PaymentOp(B, "Code:Issuer", <amount>)`
+  6. (Optional) Lock the issuing account: `SetOptionsOp(MasterWeight: 0)`
+
+Note that without this final step, there are concerns around both immutability and supply: an account may modify its NFT metadata pointer or issue more tokens unless it's locked down. Thus, Step 6 is recommended.
+
+Ownership of the token directly conveys ownership of a unit of the NFT, however that may be defined. Other scenarios such as ownership transfers via payments are built on top of this basic issuance model and are thus extraneous to this SEP.
+
+Other issuance models may exist and may be compatible with the following specification, but this is the standard issuance model on Stellar.
+
+### NFT Specification
+This specification is partially inspired by Ethereum's [EIP-721 specification](https://eips.ethereum.org/EIPS/eip-721#specification) on NFTs; however, it is generalized to arbitrary asset types (i.e. not necessarily images).
+
+In order to minimize the costs of on-chain storage, we have multiple levels of indirection of NFT metadata. The only thing the NFT account stores on-chain is a description of where to find the metadata.
+
+#### NFT Account Description
+A proper NFT account should specify its `homedomain`. It should also configure the asset in its domain's `stellar.toml` file (see [SEP-1](https://stellar.org/sep-1#currency-documentation)) if appropriate.
+
+There are two **required** fields on the account itself:
+
+| Field       | Purpose |
+| ----------- | ------- |
+| `type`      | Describes how the "first hop" of metadata is defined, standard values include `ipfshash` and `uri` |
+| `hash`      | The [SHA3](https://csrc.nist.gov/publications/detail/fips/202/final) hash of the metadata contents |
+
+
+##### `type`
```

Comment:
Actually, the big advantage to not using something like an nft namespace is the interoperability with tokens that maybe aren't conceptually NFTs but are otherwise compatible. This could also be seen as a disadvantage, but I think it's an advantage.